WebStore/Startup.cs

```diff
@@ -12,10 +13,12 @@ public class Startup
         private readonly IConfiguration _Configuration;
         public Startup(IConfiguration Configuration)
         {
-
+            _Configuration = Configuration;
         }
         public void ConfigureServices(IServiceCollection services)
         {
+            services.AddMvc().AddRazorRuntimeCompilation();
+            services.AddSingleton<DbInMemory>();
```

Comment:
Why singleton?

Reply:
Because the object will not change its state